plugins/aea-test-autonomy/aea_test_autonomy/docker/registries.py

```diff
@@ -34,7 +34,6 @@
 DEFAULT_HARDHAT_ADDR = "http://127.0.0.1"
 DEFAULT_HARDHAT_PORT = 8545
 REGISTRIES_CONTRACTS_DIR = "autonolas-registries"
-CONFIG_FILE = "hardhat.config.ts"
```

Comment:
Was not used. Also, the constants below are currently not used but might be useful in the future.